src/Razor/src/Microsoft.AspNetCore.Razor.ProjectEngineHost/CompilationTagHelperResolver.cs

```diff
@@ -10,35 +10,34 @@
 using System.Threading.Tasks;
 using Microsoft.AspNetCore.Razor;
 using Microsoft.AspNetCore.Razor.Language;
+using Microsoft.AspNetCore.Razor.PooledObjects;
 using Microsoft.AspNetCore.Razor.Serialization;
 using Microsoft.AspNetCore.Razor.Telemetry;
 using Microsoft.CodeAnalysis;
 using Microsoft.CodeAnalysis.Razor;
 
 namespace Microsoft.AspNetCore.Razor;
 
-internal class CompilationTagHelperResolver
+internal class CompilationTagHelperResolver(ITelemetryReporter? telemetryReporter)
 {
-    private readonly ITelemetryReporter? _telemetryReporter;
+    private readonly ITelemetryReporter? _telemetryReporter = telemetryReporter;
 
-    public CompilationTagHelperResolver(ITelemetryReporter? telemetryReporter)
-    {
-        _telemetryReporter = telemetryReporter;
-    }
```

Comment:
So we're just going to assume that everyone likes primary constructors are we? 😛